src/hooks/useUniswapPools.ts

```diff
@@ -148,10 +162,13 @@ export const calculateLPTokenPrice = (data: any, ethPrice: any) => {
  * @param {*} valueNow
  * @param {*} value24HoursAgo
  */
-export const getPercentChange = (valueNow: any, value24HoursAgo: any) => {
+export const getPercentChange = (
+  valueNow: string | number,
+  value24HoursAgo: string | number
+) => {
   const adjustedPercentChange =
-    ((parseFloat(valueNow) - parseFloat(value24HoursAgo)) /
-      parseFloat(value24HoursAgo)) *
+    ((parseFloat(valueNow as string) - parseFloat(value24HoursAgo as string)) /
```

Comment:
Maybe just add `String(...)` to make sure it's a string instead of lying to TS to silence it?

```suggestion
    ((parseFloat(String(valueNow)) - parseFloat(String(value24HoursAgo))) /
```